README.md

```diff
@@ -178,33 +178,27 @@ fork {
     int emiratesPrice;
 
     // Get the response and price for airline 'Qatar Airways'
-    if (airlineResponses["qatarWorker"] != null) {
-        var resQatar = check <http:Response>(airlineResponses["qatarWorker"]);
-        jsonFlightResponseQatar = check resQatar.getJsonPayload();
-        match jsonFlightResponseQatar.Price {
-            int intVal => qatarPrice = intVal;
-            any otherVals => qatarPrice = -1;
-        }
+    var resQatar = check <http:Response>(airlineResponses["qatarWorker"]);
+    jsonFlightResponseQatar = check resQatar.getJsonPayload();
+    match jsonFlightResponseQatar.Price {
+        int intVal => qatarPrice = intVal;
+        any otherVals => qatarPrice = -1;
```

Comment:
This is the use case of but operation? Shall we try that?

Reply:
Ack; Addressed as suggested.